spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/ReadIntegrationTestBase.java

```diff
@@ -364,4 +364,43 @@ public void testArrowCompressionCodec() {
 
     assertThat(avroResults).isEqualTo(arrowResultsForLZ4FrameCodec);
   }
+
+  @Test
+  public void testStringExpressions() {
```

Comment:
Where are we making the call to `BigQueryConnectorUtils.enablePushdown` so that we push down the query?

Reply:
Added a new test suite and calling `BigQueryConnectorUtils.enablePushdown` in its constructor